src_files.yml

```diff
@@ -6,9 +6,8 @@ common_cells_all:
     # package. Files in level 1 only depend on files in level 0, files in level 2 on files in
     # levels 1 and 0, etc. Files within a level are ordered alphabetically.
     # Level 0
-    - src/addr_decode.sv
-    - src/cdc_2phase.sv
     - src/cf_math_pkg.sv
+    - src/cdc_2phase.sv
```

Comment:
Same here.

Reply:
ok I undo this. I just believe pkgs should always be first